src/App/ResetCryptoStore.ts

```diff
@@ -0,0 +1,89 @@
+import { rm } from "fs/promises"
```

Comment:
N.B. Unwritten rule here is `App` is for long running processes and we have a scripts/ directory for scripts.

Reply:
Since this is a TS file, I'd like to run it with `ts-node` as is done for some of our other yarn scripts do, but most of them don't work.  For example, here's what I get from `yarn validate-config`:
```
yarn run v1.22.19
$ ts-node src/Config/Config.ts
<PATH_TO_HOOKSHOT>/matrix-hookshot/node_modules/ts-node/src/index.ts:944
          throw new TypeError(`${relative(cwd, fileName)}: Emit skipped`);
                ^
TypeError: ../matrix-bot-sdk/lib/index.js: Emit skipped
    at getOutput (<PATH_TO_HOOKSHOT>/matrix-hookshot/node_modules/ts-node/src/index.ts:944:17)
    at Object.compile (<PATH_TO_HOOKSHOT>/matrix-hookshot/node_modules/ts-node/src/index.ts:1243:30)
    at Module.m._compile (<PATH_TO_HOOKSHOT>/matrix-hookshot/node_modules/ts-node/src/index.ts:1370:30)
    at Module._extensions..js (node:internal/modules/cjs/loader:1209:10)
    at Object.require.extensions.<computed> [as .js] (<PATH_TO_HOOKSHOT>/matrix-hookshot/node_modules/ts-node/src/index.ts:1374:12)
    at Module.load (node:internal/modules/cjs/loader:1033:32)
    at Function.Module._load (node:internal/modules/cjs/loader:868:12)
    at Module.require (node:internal/modules/cjs/loader:1057:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (<PATH_TO_HOOKSHOT>/matrix-hookshot/src/Config/Config.ts:3:1)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Is there some prerequisite step needed for `ts-node` scripts to work?

Reply:
Errrm, no. I've never seen that before :worried: . Is this related to our fork?

Reply:
No; it turns out that Hookshot's `ts-node` setup is improper.  It's broken due to `scripts/.*ts` files not being in the `tsconfig.json`'s `rootDir`.

See: https://github.com/TypeStrong/ts-node/discussions/1595

I gave a shot at fixing it, but it's not quite taking.  This same issue affects `scripts/definition-fixer.ts` too, so IMO this is worth fixing in a separate changelist, as long as it's fine for the crypto reset script to stay where it is for now (note that even after being moved, it would be invoked by the same `yarn` command as it is now).

Reply:
Yeah, given we alias a yarn command I'm not super hung up on it. Though we should fix the improper setup :)

Reply:
https://github.com/matrix-org/matrix-hookshot/issues/722